uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less

```diff
@@ -549,6 +549,10 @@
                                     left: 3px;
                                     position: relative;
                                     top: 0;
+                                    /* increase clickable size of dropdown toggle */
+                                    padding: 0 5px 5px;
+                                    margin: 0 -5px -5px;
+
```

Comment:
This change is inside a `@media (min-width: 769px)`;  are the drowdown toggles large enough on displays under 769px?